diff --git a/configure.ac b/configure.ac
index 98e6e0a..951a90c 100644
--- a/configure.ac
+++ b/configure.ac
@@ -20,7 +20,7 @@ AS_IF([test "x$with_alsa" = "xno"], [
     use_alsa=0
 ], [
     use_alsa=1
-    PKG_CHECK_MODULES(DEPS, [alsa])
+    # PKG_CHECK_MODULES(DEPS, [alsa])
 ])
 AC_DEFINE_UNQUOTED([USE_ALSA], [$use_alsa],
                    [Define to 1 to enable ALSA support])
@@ -32,7 +32,7 @@ AS_IF([test "x$with_pulseaudio" = "xno"], [
     use_pulseaudio=0
 ], [
     use_pulseaudio=1
-    PKG_CHECK_MODULES(DEPS, [libpulse-simple])
+    # PKG_CHECK_MODULES(DEPS, [libpulse-simple])
 ])
 AC_DEFINE_UNQUOTED([USE_PULSEAUDIO], [$use_pulseaudio],
                    [Define to 1 to enable pulseaudio support])
@@ -57,7 +57,7 @@ AS_IF([test "x$with_sndfile" = "xno"], [
     use_sndfile=0
 ], [
     use_sndfile=1
-    PKG_CHECK_MODULES(DEPS, [sndfile])
+    # PKG_CHECK_MODULES(DEPS, [sndfile])
 ])
 AC_DEFINE_UNQUOTED([USE_SNDFILE], [$use_sndfile],
                    [Define to 1 to enable sndfile support])
